adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala

```diff
@@ -202,6 +202,18 @@ object ReferenceRegion {
     ReferenceRegion(variant.getContigName, variant.getStart, variant.getEnd)
   }
 
+  /**
+   * Builds a referenceRegion from genomic coordinates.
+   *
+   * @param referenceName reference name
+   * @param start start position
+   * @param end end position
+   * @return Reference Region for these genomic coordinates
+   */
+  def fromGenomicRange(referenceName: String, start: Long, end: Long): ReferenceRegion = {
```

Comment:
Is this method necessary?  In other words, is it difficult to call `ReferenceRegion(referenceName, start, end)` from Python?

Reply:
you cannot call ReferenceRegion.apply(referenceName, start, end) from python.

Reply:
ok